de.js

```diff
@@ -0,0 +1,334 @@
+    seed_phrase_key: `Deine Seed-Phrase ist der Schlüssel zu deiner Wallet. Jeder mit dieser Wortfolge kann auf dein Guthaben zugreifen. Tippe hier um mehr zu erfahren.`,
+    written_down: `Ich habe alles aufgeschrieben`,
+    screenshot_alert: `Screenshots oder andere digitale Kopien deiner Seed-Phrase könnten in fremde Hände gelangen`,
+    seed_phrase_access: ` Jeder mit deiner Seed-Phrase kann auf dein Guthaben zugreifen. Teile deine Seed-Phrase mit niemandem. MyDoge speichert deine Seed-Phrase nicht für dich. Deine Seed-Phrase ist die einzige Möglichkeit deine Wallet wiederherzustellen solltest du dein Gerät jemals verlieren, es beschädigt oder gestohlen werden, oder in sonstiger Form kein Zugriff mehr darauf bestehen.`,
+    icloud_backup: `iCloud-Backup`,
+    cloud_backup: `Cloud-Backup`,
+    dont_lose_wallet: `Verliere deine Wallet nicht!`,
+    save_online_backup: `Speichere ein Online-Backup per`,
+    icloud: `iCloud`,
+    google_drive: `Google Drive`,
+    do_this_later: `Ich mache das später`,
+    confirm_seed_phrase: `Seed-Phrase bestätigen`,
+    select_correct_word: `Wähle aus der Liste unten für jede Position das korrekte Wort aus.`,
+    enter_seed_phrase: `Seed-Phrase eingeben`,
+    enter_seed_phrase_instructions: `Gib deine 12-Wort-Seed-Phrase ein um deine bestehende Wallet wiederherzustellen`,
+    paste: `Einfügen`,
+    restore_wallet: `Wallet wiederherstellen`,
+    wallet_imported: `Wallet importiert!`,
+    wallet_imported_body: `Deine Wallet ist einsatzbereit.`,
+    restore_existing_wallet: `Deine bestehende Wallet wiederherstellen`,
+    restore_existing_wallet_instructions: ` Du hast also bereits eine Wallet? Dann lass sie uns wiederherstellen!`,
+    restore_from: `Wiederherstellen per`,
+    restore_from_cloud_description: `Wiederherstellen deiner Wallet per`,
+    restore_with_seed_phrase: `Wiederherstellen mit einer Seed-Phrase`,
+    restore_with_seed_phrase_description: `Nutze deine Seed-Phrase um eine bestehende MyDoge-Wallet wiederherzustellen`,
+    no_cloud_backup: `Kein Cloud-Backup`,
+    no_cloud_backup_body: `Kein Cloud-Backup gefunden. Bitte per Seed-Phrase wiederherstellen. Oder erstelle eine neue Wallet`,
+    create_wallet: `Eine Wallet erstellen`,
+    already_have_wallet: ` Ich habe bereits eine Wallet`,
+    buy_send_receive_dogecoin: `Kaufe, Sende & Empfange Dogecoin`,
+    send_doge_tips: `Versende Doge-Tips auf 𝕏, verwahre deine Doge sicher und sei deine eigene Bank`,
+    shop_gift_cards_apparel: `Kaufe Geschenkkarten und Kleidung`,
+    buy_gift_cards: `Kaufe Geschenkkarten mit Dogecoin oder shoppe die neueste Doge-Mode`,
+    enter_the_dogeverse: `Betritt das Dogeverse`,
+    claim_dogtag: `Sichere dir dein DogeTag, finde Freunde, erhalte Zahlungen über deinen Link und teile Memes`,
+    so_much_tip: `So Tip, Much Wow!`, //leave it in english I`d say^^ oder `Wow, soviel Trinkgeld`
```

Comment:
Remove comment & leave it in English:

```suggestion
    so_much_tip: `So Tip, Much Wow!`,
```
or remove comment & say it in German: 

```suggestion
    so_much_tip: `Wow, so viel Trinkgeld!`,
```

Reply:
Will leave english